### PR TITLE
Add rubocop-packaging as a known extension in the docs

### PR DESCRIPTION
**Pull request by utkarsh2102**
This PR adds another known extension, [rubocop-packaging](https://github.com/utkarsh2102/rubocop-packaging) to the [extension docs](https://docs.rubocop.org/rubocop/0.85/extensions.html).

And lastly, thanks @bbatsov, for this wonderful tool! :heart: 

Signed-off-by: Utkarsh Gupta <<utkarsh@debian.org>>

-----------------

Before submitting the PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [x] Added an entry to the [Changelog](https://github.com/rubocop-hq/rubocop/blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](https://github.com/rubocop-hq/rubocop/blob/master/CONTRIBUTING.md#changelog-entry-format).
* [x] The PR relates to *only* one subject with a clear title and description in grammatically correct, complete sentences.
* [x] Run `bundle exec rake default`. It executes all tests and RuboCop for itself, and generates the documentation.

[1]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 * [#8113](https://github.com/rubocop-hq/rubocop/pull/8113): Let `expect_offense` templates add variable-length whitespace with `_{foo}`. ([@eugeneius][])
 * [#8148](https://github.com/rubocop-hq/rubocop/pull/8148): Support autocorrection for `Style/MultilineTernaryOperator`. ([@koic][])
 * [#8151](https://github.com/rubocop-hq/rubocop/pull/8151): Support autocorrection for `Style/NestedTernaryOperator`. ([@koic][])
+* [#8160](https://github.com/rubocop-hq/rubocop/pull/8160): Add `rubocop-packaging` as a known extension in the docs. ([@utkarsh2102][])
```
Comment (bbatsov) [inline CHANGELOG.md:11]: We typically don't mention doc updates in the changelog.

**Comment by bbatsov** [comment]
Cool package! I've always been fond of Debian! :-)

> And lastly, thanks @bbatsov, for this wonderful tool! ❤️

You're welcome! 🙇‍♂️

File: CHANGELOG.md (hunk 1 above)
Comment (utkarsh2102) [reply CHANGELOG.md]: Ah, apologies. Fixed! :smile:

**Comment by utkarsh2102** [comment]
> Cool package! I've always been fond of Debian! :-)

Awesome, thanks! :smile:

**Comment by utkarsh2102** [comment]
Hi @bbatsov 👋🏻 

Once again, thanks for all your work so far! ❤️ 

I have got a small request to make..

Based on rspec-style-guide, minitest-style-guide, rails-style-guide, and so on, I've created packaging-style-guide, here:
https://github.com/utkarsh2102/packaging-style-guide
(it follows the same style like other guides)

Would you be kind enough to give a little space to host it at rubystyle.guide? So like https://{rspec, minitest, rails}.rubystyle.guide, there'll be https://packaging.rubystyle.guide, too? 

Please let me know if there's something more required at my end to make this happen?
It'd really mean a lot to us (the Debian core devs and other d/s maintainers) ❤️

**Comment by bbatsov** [comment]
I think you'll have to transfer this guide to the RuboCop HQ org (you'll still be it's owner), so I can deploy the HTML to the org's GitHub pages like the other guides.

**Comment by utkarsh2102** [comment]
Aha, okay. As long as I can keep maintaining this, this won't be a problem!
However, whilst trying to transfer this to the RuboCop HQ org, I get this:
` You don’t have the permission to create public repositories on rubocop-hq`

How do I proceed from here?

**Comment by bbatsov** [comment]
I've sent you an invite to join the org. You'll be able to transfer the repo after you accept it.

**Comment by utkarsh2102** [comment]
Many thanks. The transfer is now done! 🎉

**Comment by bbatsov** [comment]
@utkarsh2102 See http://packaging.rubystyle.guide :-)

**Comment by utkarsh2102** [comment]
@bbatsov, you're the best, thank you! ❤️

**Pull request merged**